t/torchtext/build_info.json

```diff
@@ -0,0 +1,16 @@
+{
+  "maintainer": "aastha-sharma2",
+  "package_name": "torchtext",
+  "github_url": "https://github.com/pytorch/text.git",
+  "version": "0.6.0",
+  "wheel_build" : true,
+  "package_dir": "t/torchtext",
+  "default_branch": "main",
+  "build_script": "torchtext_ubi_9.3.sh",
+  "docker_build": false,
+  "validate_build_script": true,
+  "use_non_root_user": "false",
+  "*.*.*": {
```

Comment:
Can't we simply set it "*", why to complicate?

Reply:
> Can't we simply set it "*", why to complicate?

PR ready for review now